frontend/core-ui/src/modules/settings/structure/components/units/UnitsColumns.tsx

```
@@ -123,9 +124,10 @@ export const UnitsColumns: ColumnDef<IUnitListItem>[] = [
     cell: ({ cell }) => {
       return (
         <RecordTableCellDisplay>
-          <SelectDepartment
-            value={cell.getValue() as string}
-            onValueChange={() => {}}
+          <SelectDepartmentTree
+            recordId={cell.id}
+            selected={cell.getValue() as string}
+            onSelect={() => {}}
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)